web/cypress/integration/upstream/create_upstream_with_retry_timeout.spec.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create Upstream With retry_timeout', () => {
+  const selector = {
+    name: '#name',
+    nodes_0_host: '#nodes_0_host',
+    nodes_0_port: '#nodes_0_port',
+    nodes_0_weight: '#nodes_0_weight',
+    retry_timeout: '#retry_timeout',
+    input: ':input',
+    notification: '.ant-notification-notice-message',
+    nameSelector: '[title=Name]',
+    upstreamType: '.ant-select-item-option-content',
+    drawer: '.ant-drawer-content',
+    monacoScroll: '.monaco-scrollable-element',
+    description: '#desc',
+  };
+  const data = {
+    upstreamName: 'test_upstream',
+    description: 'desc_by_autotest',
+    ip1: '127.0.0.1',
+    createUpstreamSuccess: 'Create Upstream Successfully',
+    deleteUpstreamSuccess: 'Delete Upstream Successfully',
+    port0: '7000',
+    weight0: '2',
+    port1: '7001',
+    weight1: '2',
+    retry_timeout: '3',
```

Comment:
since `retry_timeout ` has been defined, we can use it in the Line60, the same goes for other test data